frontend/src/components/DeleteAccountAlert.js

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Slide from '@mui/material/Slide';
+import { TransitionProps } from '@mui/material/transitions';
+import axios from "axios";
+import { useSessionStorage } from "../customHooks";
+import { STATUS_CODE_FORBIDDEN, STATUS_CODE_OK, STATUS_CODE_UNAUTHORIZED } from "../constants";
+import { URL_USER_SVC } from "../configs";
+import { useNavigate } from "react-router-dom";
+
+const Transition = React.forwardRef(function Transition(
+    props: TransitionProps & {
+        children: React.ReactElement<any, any>;
+    },
+    ref: React.Ref<unknown>,
+) {
+    return <Slide direction="up" ref={ref} {...props} />;
+});
```

Comment:
These are Typescript code, should use Javascript version

Reply:
My bad, was working under high fever and for unknown reasons I did that. Converted to Javascript 😊